cognite/neat/_constants.py

```diff
@@ -200,6 +201,22 @@ def get_asset_read_only_properties_with_connection() -> "list[DMSProperty]":
     return [DMSProperty.model_validate(item) for item in (_ASSET_ROOT_PROPERTY, _ASSET_PATH_PROPERTY)]
 
 
+def get_base_concepts(
+    base_model: Literal["CogniteCore"] = "CogniteCore",
+    total_concepts: int | None = None,
+) -> list[str]:
+    """Gets the base concepts for a given base model represented in the short form.
+    Args:
+        base_model: The base model to get the concepts for.
+        total_concepts: The number of concepts to get. If None, all concepts are returned.
+    """
+
+    if base_model == "CogniteCore":
+        return [f"cdf_cdm:{concept}(version=v1)" for concept in COGNITE_CONCEPTS][:total_concepts]
```

Comment:
`COGNITE_CONCEPT` constant should be sorted somehow, so the list slicing here to be a bit more predictable, wdyt?